courses/intro-to-spark/book/03_Proof_Of_Program_Integrity.rst

```diff
@@ -9,43 +9,43 @@ supporting SPARK. In this section, we focus on the simpler proofs that are
 needed for program integrity.
 
 For its soundness, this analysis relies on the prerequisite of having performed
-flow analysis and a user should not proceed further if there are still
-unanswered flow analysis messages for their program. The primary objective of
+:ref:`Flow Analysis` and a user should not proceed further if there are still
+unjustified flow analysis messages for her program. The primary objective of
```

Comment:
Maybe we should change to "users should not... [...] for their program". It's gender-neutral and quite common nowadays.

Reply:
There are other occurrences where we talk of one user, and I find it odd to try each time to avoid the gender issue. I'd rather keep it as it is.